parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -144,6 +150,10 @@ String typeParametersAsString() {
     return "";
   }
 
+  public boolean isValidColumnOrder(ColumnOrder columnOrder) {
```

Comment:
I think, we don't need it public.

Reply:
Indeed, we don't, package protected is sufficient.